ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+        }
+
+        static void EmptyArray()
+        {
+            var arr = new string[] { };
+            Console.WriteLine(arr);
+        }
+        static void OneDimensionalArray()
+        {
+            object[] arr2 = new object[3] { 32, 'A', "Hello" };
+
+            for (int i = 0; i < 3; i++)
+            {
+                Console.WriteLine(arr2[i]);
+            }
+
+            int intBuffer = Convert.ToInt32(arr2[0]);
+            intBuffer += 10;
+            arr2[0] = intBuffer;
+            string stringBuffer = Convert.ToString(arr2[2]);
+            stringBuffer += " guys!";
+            arr2[2] = stringBuffer;
+
+            for (int i = 0; i < 3; i++)
+            {
+                Console.WriteLine(arr2[i]);
+            }
+        }
+        static void Max13Array()
+        {
+            Random rand = new Random();
+            int arr3buffer = 0;
+            int[] arr3 = new int[13];
+            for (int i = 0; i < arr3.Length - 1; i++)
+            {
+                arr3[i] = rand.Next(0, 50);
+            }
+            for (int i = 0; i < arr3.Length - 1; i++)
+            {
+                if (arr3[i] > arr3buffer)
+                {
+                    arr3buffer = arr3[i];
+                }
+            }
+            for (int i = 0; i < arr3.Length - 1; i++)
+            {
+                Console.WriteLine(arr3[i]);
+            }
+            Console.WriteLine($"Макс. знач: {arr3buffer}");
+        }
+        static void boxingunboxing()
+        {
+            short i1 = 426;
+            object i2 = i1;
+            sbyte i21 = (sbyte)i2;
```

Comment:
incorrect a bit. You forgot the main rule of unboxing. And as a result, your code is handling an exception:

![image](https://user-images.githubusercontent.com/3023032/108840977-20e9fc80-75e8-11eb-9b40-1ffb9da8031c.png)

Reply:
![image](https://user-images.githubusercontent.com/3023032/108841142-5b539980-75e8-11eb-81ad-2cf46f0c76d2.png)